docs/Manifest.toml

```diff
@@ -19,11 +19,11 @@ version = "0.8.1"
 
 [[Documenter]]
 deps = ["Base64", "Dates", "DocStringExtensions", "InteractiveUtils", "JSON", "LibGit2", "Logging", "Markdown", "REPL", "Test", "Unicode"]
-git-tree-sha1 = "2b45ba82d7de1083af18f09af4ebdeb4dd99b9f3"
```

Comment:
@DilumAluthge, I don't think these unrelated manifests should get changed.

Reply:
https://github.com/KristofferC/PackageCompilerX.jl/pull/50 will disable the "Manifest update" feature of CompatHelper